tests/trac/requirements.txt

```diff
@@ -0,0 +1,3 @@
+passlib
+trac == 1.6
+trac_ticketrpc == 0.9.3
```

Comment:
Please add this directory to https://github.com/kiwitcms/trackers-integration/blob/main/.github/dependabot.yml